src/Kdyby/Translation/LocaleResolver/AcceptHeaderResolver.php

```diff
@@ -52,7 +53,42 @@ public function resolve(Kdyby\Translation\Translator $translator)
 			return substr($locale, 0, 2);
 		}, $translator->getAvailableLocales());
 
-		return $this->httpRequest->detectLanguage($short) ?: NULL;
+		return $this->detectLanguage($short, $header) ?: NULL;
```

Comment:
I previously added the `?: NULL` part [here](https://github.com/Kdyby/Translation/pull/7). I presume that detectLangage can sometimes return an empty string (I would not have added it without reason ant the detectLanguage method was not changed since). Which means it probably is still necessary.